packages/contracts/src/regenesis/make-regenesis.ts

```diff
@@ -0,0 +1,36 @@
+export const makeRegenesisDump = (dump:any, genesis: any) => {
```

Comment:
This is the script that we have used to make the regenesis: https://github.com/ethereum-optimism/scripts/blob/main/scripts/state-surgery.js. It currently includes some specific logic for the previous regenesis, so look at its history if you are interested in seeing a more pure form.

Whenever there is another regenesis, we will move it into the monorepo and be very thorough around using it and documenting it.